Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7.2-alpine3.8
+
+WORKDIR /opt/slack-export-viewer
+
+COPY requirements.txt ./
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+COPY . /
+
+EXPOSE 5000
+
+RUN ["chmod", "+x", "/entrypoint.sh"]
+ENTRYPOINT ["sh", "/entrypoint.sh"]
```

Comment:
```suggestion
CMD ["/entrypoint.sh"]
```

Reply:
entrypoint is generally the interpreter. E.g. `perl`, `bash`, `node`

Reply:
`CMD` is the argument passed to the entrypoint

Reply:
does not work with cmd

Reply:
i've been finding how to pass agrs from console when run

Reply:
ok, try `CMD ["-c", "/entrypoint.sh"]` instead

Reply:
or `CMD /entrypoint.sh`

Reply:
does not work